Common/NearestNeighbour/src/main/java/org/tribuo/common/nearest/KNNTrainer.java

```diff
@@ -132,11 +137,7 @@ public KNNTrainer(int k, DistanceType distType, int numThreads, EnsembleCombiner
      */
     @Deprecated
     public KNNTrainer(int k, Distance distance, int numThreads, EnsembleCombiner<T> combiner, Backend backend) {
-        this.k = k;
-        this.distType = distance.getDistanceType();
-        this.numThreads = numThreads;
-        this.combiner = combiner;
-        this.backend = backend;
+        this(k, distance.getDistanceType(), numThreads, combiner, backend);
```

Comment:
Similarly this doesn't need to call postConfig because the base constructor does.